modules/feature-upload.php

```diff
@@ -291,7 +304,7 @@
 				exit();
 			}
 			
-			$filepath = $env->storage_prefix . $pageindex->{$env->page}->uploadedfilepath;
+			$filepath = getcwd() . "/" . $env->storage_prefix . $pageindex->{$env->page}->uploadedfilepath;
```

Comment:
Too true.  Maybe we should check to see if the dir exists as-is, call getcwd() if it doesn't, then check to see if _that_ exists, and if it still isn't right, throw an error?